src/playbackstrategy/modifiers/html5.js

```diff
@@ -373,7 +366,14 @@ function Html5 () {
         }
       }
     }
-    return undefined
```

Comment:
Just wondering if leaving out this return undefined was deliberate?

Reply:
Yes, the function will return undefined.